packages/dashboard/src/helpers/filterData.ts

```diff
@@ -35,7 +35,12 @@ function filter(data = [], filters: SharedFilter[], condition) {
         isNotTheSelectedValue = selectedValue && currentValue != selectedValue
       }
 
-      if ((filter.filterStyle === 'nested-dropdown' && filter.subGrouping && filter.active === currentValue, isNotTheSelectedValue === false)) {
+      if (
```

Comment:
@Matt-DiPaolo can you make sure this is right? it was resolving to always true otherwise.

Reply:
I may be misunderstanding, but it is not always resolving to true. It is only true when:
- selectedValue is not an array
- selectedValue is undefined
- currentValue doesn't equal selectedValue

Since nested dropdowns is working with a subgroup. The currentValue and the selected Value can equal each other. e.g. **Age Group - 56 years old** and **Age Group - Overall** have the same currentValue and selectedValue of "Age Group." This example would set isNotTheSelectedValue to false.

If it is a nested dropdown and the isNotTheSelectedValue is false, then set isNotTheSelectedValue to true if:
- subGroupActive is undefined 
- subGroupActive doesn't equal the selectedSubGroupValue

I am going to remove the 
 `
if (subGroupActive === undefined || value !== subGroupActive) {
          isNotTheSelectedValue = true
        } 
`
and replace with 
` isNotTheSelectedValue = subGroupActive && selectedSubGroupValue !== subGroupActive`